### ISSUE TITLE
Traceback: Missing dependencies for breakable, break

### ISSUE THREAD

**Issue by saraswat** [epfl-lara/stainless#771]
Ok this one is a Sev 2 from my vantage point.

Stainless does not support `return`. One way out -- when attempting to get existing (Scala) code to work with Stainless without significant rewriting -- is to use `breakable` and `break`. 

But Stainless does not understand them, as the code below demonstrates. 

Would be great if this could be fixed.

The code:
```
import stainless.annotation._
import scala.util.control.Breaks.{break, breakable} 
object test {
  def m(x:Int)  = {
    var result: Int = 0
    breakable {
      if (x > 0) {
        result = 1
        break
      }
    }
    result
  }
}

```
results in the traceback:
```
error] m$0 depends on missing dependencies: breakable$0, break$0.
[error] Cannot recover from missing dependencies
[error] ## Exception when compiling 31 sources to /Users/savija/tmp/test/verified/target/scala-2.12/classes
[error] inox.package$FatalError: Cannot recover from missing dependencies
[error] inox.Reporter.onFatal(Reporter.scala:39)
[error] inox.Reporter.fatalError(Reporter.scala:50)
[error] inox.Reporter.fatalError(Reporter.scala:98)
...
```

**Comment by saraswat** [comment]
Would appreciate any work-arounds in the meantime...

**Comment by romac** [comment]
I am afraid I don't see a way to support such code without rewriting it at the moment.

There might be a way to define alternatives to `breakable` and `break` in terms of a continuation monad [1] but it's unclear how well this would work, especially when combined with stateful mutations which would have to be lifted in the monad as well and thus require even more code changes.

[1] See the `Abortable` type in the `Maybe` section of https://blog.poisson.chat/posts/2019-10-26-reasonable-continuations.html

**Comment by jad-hamza** [comment]
No, only `return` for now. I think @romac idea was that, eventually, the `ControlFlow` case class that we use for return could be extended for a case for `break`.
